traffic_ops/traffic_ops_golang/test/error_context_test.go

```diff
@@ -0,0 +1,162 @@
+/*
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package test
+
+import (
+	"fmt"
+	"testing"
+)
+
+var codes []uint
+
+func init() {
```

Comment:
`init` is dangerous and unintuitive, because people importing your package don't necessarily know what's happening or when. It should be avoided when it isn't necessary. 

It isn't necessary here, you can 
```
var codes =[]uint{
	0, 1, 2,
}
```